packages/app/src/debug/empty/DebugEmptyStates.cy.tsx

```diff
@@ -84,7 +84,7 @@ describe('Debug page empty states', () => {
 
       mountWithGql(<DebugNotLoggedIn />)
 
-      cy.findByRole('link', { name: 'Learn about debugging CI failures in Cypress' }).should('have.attr', 'href', 'https://on.cypress.io/debug-page?utm_source=Binary%3A+Launchpad&utm_medium=Debug+Tab&utm_campaign=Learn+More')
+      cy.findByRole('link', { name: 'Learn more' }).should('have.attr', 'href', 'https://on.cypress.io/debug-page?utm_source=Binary%3A+Launchpad&utm_medium=Debug+Tab&utm_campaign=Learn+More')
```

Comment:
How come these onlinks have query params hard-coded? these might go stale if these headings ever change in the docs